html/changelogs/Deadgebert-AddIceboxMagboots.yml

```diff
@@ -0,0 +1,58 @@
+################################
+# Example Changelog File
```

Comment:
thats not how you add changelogs. they are automatically done from what you put in the pr body
remove this file

Reply:
ah sorry, done, thanks